src/commands/mod.rs

```diff
@@ -35,6 +44,10 @@ impl Command {
             Self::Nsec3Hash(nsec3hash) => nsec3hash.execute(env),
             Self::Key2ds(key2ds) => key2ds.execute(env),
             Self::Help(help) => help.execute(),
+            Self::Report(s) => {
+                write!(env.stdout(), "{s}");
```

Comment:
Do we want to use `writeln!()` here?  Especially for one-line reports, nobody remembers to put a newline after.